drools-wb-screens/drools-wb-drl-text-editor/drools-wb-drl-text-editor-backend/src/test/resources/org/drools/workbench/screens/drltext/backend/server/drl/src/main/java/org/kiegroup/NumericalTypes.java

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright 2018 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.kiegroup;
+
+public class NumericalTypes {
+
+    private java.math.BigDecimal bigDecimalField;
+    private java.math.BigInteger bigIntegerField;
+    private Byte byteField;
+    private Double doubleField;
+    private Float floatField;
+    private Integer integerField;
+    private Long longField;
+    private Short shortField;
+    private String result;
+
+    public NumericalTypes() {
+    }
+
+    public NumericalTypes(
```

Comment:
Not sure if our check-styles plugin will moan about the constructor parameters not starting on the same line as the opening parenthesis but let's wait and see (AFAIK our code styles have the first parameter on the same line as the opening parenthesis).

Reply:
@manstis, I fixed it, I don't want your preferences to be violated either. :-)